docs/walkthrough/addNewDashboardPanel.md

```diff
@@ -0,0 +1,78 @@
+Walkthrough: Adding a New Panel to the Dashboard
+================================================
+
+The following is for creating a new panel called "**examplePanel**."
```

Comment:
Do you think it might be handy to explain what a panel is in the first place? Perhaps for someone looking to customize this might already be obvious though. What are your thoughts?

Reply:
Its a dashboard panel.  I guess I could explicitly state that... but since the title says it and the code is all under the dashboard folder... it seemed redundant.

Reply:
Yeah that does. I think I was thinking more of what is a page, vs a panel vs a tab but that would probably be in the first readme not here so feel free to ignore.